frontend/keyboardNav.js

```diff
@@ -209,29 +214,32 @@ function getNewSelection(dest: Dest, store: Store): ?ElementID {
     pix = pchildren.indexOf(store._parents.get(id));
   }
   if (dest === 'prevSibling') {
-    if (pix === 0) {
```

Comment:
This looks like a messy diff, but I just moved this part *down*, and put the `else` branch into a `while` loop.